src/containers/NavContainer/NavContainer.scss

```diff
@@ -0,0 +1,39 @@
+.nav-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  height:100vh;
+  width: 21%;
+  border-top-right-radius: 20px;
+  background-color: #8AAAE5;
+
+
+  &__logo{
+    display:flex;
+    height:17.5%;
+    justify-content:center;
+    align-items:center;
+  }
+
+  &__menu-options{
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height:29%;
+  }
+  &__settings{
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    height:49.5%;
+
+  }
+  &__line-break{
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    height:4%;
```

Comment:
Use whole numbers (rounded to nearest 5) and use view height (vh) instead of percentages for reasons discussed.